rerun_js/web-viewer-react/package.json

```diff
@@ -12,7 +12,8 @@
   ],
   "scripts": {
     "build:types": "tsc --noEmit && dts-buddy",
-    "build": "npm run build:types"
+    "build": "npm run build:types",
+    "build:no-check": "npm run build"
```

Comment:
This does the same thing as `build`, it's only here for compatibility with running the same command across all packages in the yarn workspace (`yarn --cwd rerun_js workspaces run build:no-check` would fail on the `web-viewer-react` package otherwise)

Reply:
That's a great comment - it should be comited :)

Reply:
yes, it would be nice if it was possible to put comments in JSON

Reply:
oh right 🤦  God damn JSON